source/sync.txt

```diff
@@ -8,21 +12,129 @@
    :titlesonly:
    :hidden:
    
-   What is {+sync+}? </sync/learn>
+   Get Started </sync/get-started>
    Configure and Update Your Data Model </sync/configure-your-data-model>
    Define Data Access Patterns </sync/data-access-patterns>
    Configure Sync </sync/configure-device-sync>
    Handle Errors </sync/error-handling>
    Add Sync to a Local-Only App </sync/local-to-sync>
+   Production Checklist </sync/production-checklist>
    Sync Production Load Testing </sync/production-load-testing>
    Event Library </sync/event-library>
+   Technical Details </sync/details>
+
+.. introduction::
+
+   You're developing a mobile app. Your users want their data saved on the cloud
+   and accessible from their other devices. Network access on a mobile device
+   can be hit or miss, so you write data locally on the device first. A
+   background process then synchronizes the data to the cloud and resolves any
+   conflicting writes.
+ 
+   {+sync+} does all that... and more!
```

Comment:
Nit: this is a tempting way to phrase this because it's true, but it is pretty informal/casual compared to our tone throughout the rest of the docs, and also reads as a bit salesy to me. We can probably do without this line entirely.

Reply:
Fine, but the above paragraph needs a clincher, otherwise I haven't connected it to the actual topic at hand, just described a scenario.

Reply:
I think the "built for this purpose" that you've added works, or we could say something like "handles this for you." The former says "we did this thing and this is why" but the latter says "this thing solves this problem for you." It's a subtle nuance, so I don't have a strong opinion.